src/core/include/openvino/op/string_tensor_unpack.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
+namespace v15 {
+/// \brief Operator unpacking a batch of strings into three tensors.
+/// \ingroup ov_ops_cpp_api
+class OPENVINO_API StringTensorUnpack : public ov::op::Op {
+public:
+    OPENVINO_OP("StringTensorUnpack", "opset15", ov::op::Op);
+
+    StringTensorUnpack() = default;
+    /// \brief Constructs a StringTensorUnpack operation.
+    ///
+    /// \param data Input strings encoded in utf-8 bytes
```

Comment:
this should be a tensor of `ov::element::string` elements

Reply:
Good catch, thanks! Fixed.